internal/work/dispatcher.go

```diff
@@ -136,16 +139,71 @@ func (d *Dispatcher) Connect() error {
 				}
 				event.Name = ipc.WorkerEventName(eventName)
 				event.Worker = strings.TrimPrefix(dest, "com.redhat.Yggdrasil1.Worker1.")
+				workerMessageID := ""
 				switch ipc.WorkerEventName(eventName) {
```

Comment:
I recommend adding a `default` case to this switch that reports an unhandled
event type and continues the loop. You probably don't want execution to continue
when variables like `workerMessageID` are empty strings.